base/special/rem_pio2.jl

```diff
@@ -0,0 +1,310 @@
+# This file is a part of Julia. Except for the rem_pio2_kernel, and
+# cody_waite_* methods (see below) license is MIT: https://julialang.org/license
+
+# rem_pio2_kernel and cody_waite_* methods are heavily based on FDLIBM code:
+# __ieee754_rem_pio2, that is made available under the following licence:
+
+## Copyright (C) 1993 by Sun Microsystems, Inc. All rights reserved.
+##
+## Developed at SunPro, a Sun Microsystems, Inc. business.
+## Permission to use, copy, modify, and distribute this
+## software is freely granted, provided that this notice
+## is preserved.
+
+const invpio2 = 6.36619772367581382433e-01
```

Comment:
Looking at it, I might just as well change this to `2/pi` and have no constant/variable at all. I didn't do this in the beginning, but I'm doing the same in the sin/cos/sincos PR. If we need a special hi/lo split as in cody waite it won't work of course, but
```julia
julia> function f(x)
       round(x*(2/pi))
       end
f (generic function with 1 method)

julia> @code_llvm f(2.0)

define double @julia_f_62153(double) #0 !dbg !5 {
top:
  %1 = fmul double %0, 0x3FE45F306DC9C883
  %2 = call double @llvm.rint.f64(double %1)
  ret double %2
}
```